app/views/dealings/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class='transaction-page'>
+  <header class='flex f-width'>
+    <nav class='flex row f-width'>
+      <%= link_to categories_path do %>
+        <span class="material-symbols-outlined">
+          arrow_left
+        </span>
+      <% end %>
+      <h1>Transactions</h1>
+    </nav>
+  </header>
+  <div class='total-amount flex row f-width'>
+    <p><em><%= @category.name %></em></p>
+    <p>Total amount: $ <%= @total %></p>
+  </div>
+  <div class='categories-container f-width'>
+    <% if @msg_bool %>
+      <p class='temp-msg'>
+        <em>Please add a transaction</em>
+      </p>
+    <% end %>
+    <% if @category_dealings %>
+      <% @category_dealings.each do |item| %>
+        <div class='item-card flex row f-width'>
+          <div class='item-desc'>
+            <h3 class='item-header'><%= item.name %></h3>
+            <p class='item-creation-date'><%= item.created_at.strftime('%a %d %b %Y') %></p>
+          </div>
+        
+          <p class='amount'>$ <%= item.amount %></p>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div class='create-container'>
+    <%= link_to new_category_dealing_path do %>
```

Comment:
- [ ] Again, this button looks awesome. Unfortunately, you should follow the design guidelines as mentioned before.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/227736140-3e385a17-d858-49dd-b57c-e3400fa9786a.png)### Original
![image](https://user-images.githubusercontent.com/45460192/227736099-d3114ee7-88f0-4929-aa05-ab67a5f934bd.png)

Reply:
- [ ] Again here as requested by the previous reviewer, the color of the button should be green with a white border and white text and not the blue you have changed it to.

Your design
![image](https://user-images.githubusercontent.com/81932478/227741833-f67db16a-b65d-4986-9026-06ca4d351f3b.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/227741888-36ab4f58-95ee-44a1-bb8e-c6c6ea8be084.png)